src/joint.mvc.view.js

```diff
@@ -28,6 +28,20 @@ joint.mvc.View = Backbone.View.extend({
 
         this.setTheme(this.options.theme);
         this.init();
+        this._ensureElClassName();
+    },
+
+    _ensureElClassName: function() {
+
+        var className = typeof this.className === 'function' ? this.className() : this.className;
```

Comment:
It's better to do `var className = _.result(this, 'className');` here. If the `className` is a function it will be executed in `this` context (that is also what Backbone does).